src/app/modules/create-facility/pages/facility-info/facility-info.component.html

```diff
@@ -1,8 +1,7 @@
 <common-page-framework layout='blank'>
-  <h1>{{pageTitle}}</h1>
-  <p class="border-bottom">
-      To qualify a Facility must be located in British Columbia
-  </p>
+  <h1 [innerHTML]='textService.textValues.SPA_TEXT_REGISTER_FACILITY_FAC_INFO_PAGE_TITLE'>{{pageTitle}}</h1>
+  <div [innerHTML]='textService.textValues.SPA_TEXT_REGISTER_FACILITY_FAC_INFO_PAGE_DESCRIPTION'></div>
+  <hr/>
```

Comment:
Might want to change to padding in the div eg. class="pb-2 border-bottom"  this is how most of the code is done.  Again, just a suggestion for consistency.